mcs/class/System/System_test.dll.sources

```diff
@@ -199,6 +199,7 @@ System.Diagnostics/EventSourceCreationDataTest.cs
 System.Diagnostics/FileVersionInfoTest.cs
 System.Diagnostics/PerformanceCounterPermissionAttributeTest.cs
 System.Diagnostics/PerformanceCounterPermissionTest.cs
+System.Diagnostics/PerformanceCounterTests.cs
```

Comment:
The name should be `System.Diagnostics/PerformanceCounterTest.cs`, this is going to create 2 files with https://github.com/mono/mono/pull/6298 otherwise.